pkg/util/pass-through/pass_through_test.go

```diff
@@ -24,6 +19,36 @@ func TestIsPassThroughPath(t *testing.T) {
 			path: "/version",
 			verb: "get",
 			want: true,
+		}, {
+			name: "/healthz::update is not pass through path",
+			path: "/healthz",
+			verb: "update",
+			want: false,
+		}, {
+			name: "/healthz::get is pass through path",
+			path: "/healthz",
+			verb: "get",
+			want: true,
+		}, {
+			name: "/livez::create is not pass through path",
+			path: "/livez",
+			verb: "create",
```

Comment:
'create' is not an API verb; it should be 'post'.

Reply:
Fixed in [11f5fae](https://github.com/kubeedge/kubeedge/pull/5143/commits/11f5fae8821a70ba5fb242d0c825b2d5967136a7).